laa_crime_application_store_app/services/permissions.py

```diff
@@ -0,0 +1,47 @@
+from fastapi import Depends
+from fastapi_azure_auth.exceptions import InvalidAuth
+from fastapi_azure_auth.user import User
+from sentry_sdk import capture_message
+
+from laa_crime_application_store_app.config.permission_settings import get_permissions
+from laa_crime_application_store_app.models.application_schema import Application
+from laa_crime_application_store_app.services.auth_service import azure_schema
+
+permissins = get_permissions()
+
+
+def validate_can_create(user: User = Depends(azure_schema)) -> None:
+    if user is None or not user.roles:
```

Comment:
Didn't we say that we would have a check for whether or not the authentication was set here?
`settings.authentication_required.lower() != "true"`

Reply:
also would be better to not have this as an `elif`, can we just return from this if block and then have the next block as another if?

Reply:
And we won't remove this completely when the roles are implemented as we will still need to check if we have authentication turned on or not.